saviSimulation/savi/jason_processing/SAVIWorld_model.java

```diff
@@ -39,7 +39,11 @@
 	private int NUMBER_UAV;
 	private int NUMBER_UGV;
 	private double SENSORS_ERROR_PROB;
-	private double SENSORS_ERROR_STD_DEV;
+	private double SENSORS_ERROR_POSITION;
+	private double SENSORS_ERROR_COMPASS_ANGLE;
+	private double SENSORS_ERROR_AZIMUTH_DETECTED;
+	private double SENSORS_ERROR_ELEVATION_DETECTED;
+	private double SENSORS_ERROR_RANGE_DETECTED;
```

Comment:
It might be good to refactor some of the configuration variables into a separate class (i.e. Maybe Config.java). The class would be responsible for loading and managing the configuration of the program. This would help with making sure that SAVIWorld_model focuses solely on it's main task (generating the simulation model).

This doesn't have to be addressed in this PR, but would be a good consideration if we are looking to further decouple our code (& promoting high cohesion)

Reply:
I agree with the Config.java idea. I see that this has been implemented in #159.